literature.bib

```diff
@@ -10141,22 +10385,80 @@ @article{useMRKN:iFM13
 }
 
 @phdthesis{L12,
-  author = {Thomas Leich},
-  title = {{Variables Nanodatenmanagement f\"ur eingebettete Systeme}},
-  school = {University of Magdeburg},
+	author = {Thomas Leich},
+	title = {{Variables Nanodatenmanagement f\"ur eingebettete Systeme}},
+	school = {University of Magdeburg},
 	address = {Germany},
 	tt-tags = {Feature Orientation},
-  year = 2012
+	year = 2012
 }
 
 @mastersthesis{B12,
 	type = {Bachelor's Thesis},
 	author = {Benduhn, Fabian},
 	title = {{Contract-Aware Feature Composition}},
 	school = {University of Magdeburg},
-  address = {Germany},
+	address = {Germany},
 	tt-tags = {Feature Orientation,Design by Contract},
-	year = 2012
+	typo3Tags = {SupervisorTT},
+	year = 2012,
+	month = OCT
+}
+
+@mastersthesis{Lueddecke12,
+	author = {David L{\"u}ddecke},
+	title = {{Extraktion von Feature-Modellen aus Implementierungsartefakten}},
+	school = {University of Magdeburg},
+	address = {Germany},
+	typo3Tags = {SupervisorTT},
+	year = 2012,
+	month = OCT,
+	note = {In German}
+}
+
+@mastersthesis{IvonneSchroeter12,
+	author = {Ivonne Schr{\"o}ter},
+	title = {{Modulare Integration von konfigurierbaren Transaktionsmechanismen in Cloud-basierte Datenspeichersysteme}},
+	school = {University of Magdeburg},
+	address = {Germany},
+	type = {Master's Thesis (Diplomarbeit)},
+	typo3Tags = {SupervisorTT},
+	note = {In German},
+	year = 2012,
+	month = SEP
+}
+
+@mastersthesis{Holthusen12,
+	author = {S{\"o}nke Holthusen},
+	title = {{Typsicherheit in Feature-orientierten Software-Produktlinien in FeatureIDE}},
+	school = {University of Magdeburg},
+	address = {Germany},
+	typo3Tags = {SupervisorTT},
+	note = {In German},
+	year = 2012,
+	month = JUL
+}
+
+@mastersthesis{ReimarSchroeter12,
```

Comment:
key intended?

Reply:
Yes because
- S12 is already in use and
- there are two master's thesis in 2021 where both authors are called Schröter. So I could not just use the last name. I then decided to use also forenames for the keys and thus introduced: `IvonneSchroeter12` and `ReimarSchroeter12`.